lib/components/slideshow/fade.js

```diff
@@ -199,8 +200,13 @@ var Fade = function (_Component) {
 
       clearTimeout(this.timeout);
       var value = { opacity: 0 };
+      var this2 = this;
       animate();
       function animate() {
+        if (this2.willUnmount) {
```

Comment:
maybe try the `animate = () => {` was of defining the method. That way you don't need the `this2` here.